### DISCUSSION TITLE
Only 3 tasks (PC1) can be parallel. Is it normal?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9226] [General Q&A] [answered]
CPU: 3960x, memory: 256g, SSD: 2T, now only three tasks（pc1） can be parallel. Is it normal? In order to see that CPU is still free, what is the bottleneck now?

**Comment by stuberman** [comment, marked as answer]
With 256 GB RAM the most you will get is 4xPC1. 3xPC1 is better allowing more ram for other tasks. 
Run `lotus-miner sealing workers` to see how your memory is being used.
